cli/src/commands/init.ts

```
@@ -39,6 +42,18 @@ export async function initCommand(options: InitOptions): Promise<void> {
     }
   }
 
+  const onboardingConfigPath = join(dirname(config.getConfigPath()), 'onboarding.json');
+  const onboardingFlow = createOnboardingFlow(onboardingConfigPath);
+
+  const setupResult = await onboardingFlow.runInitialSetup();
+  if (!setupResult.completed && setupResult.issues && setupResult.issues.length > 0) {
+    console.log(chalk.yellow('⚠️  Onboarding completed with issues:'));
+    setupResult.issues.forEach((issue) => console.log(chalk.yellow(`  • ${issue}`)));
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)